airflow/www/static/foundation.min.css

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
Did we check license compatibility with apache for all of the third party files? This one seems to be fine (MIT).
http://apache.org/legal/resolved.html#category-a